PWGLF/TableProducer/strangenessbuilder.cxx

```diff
@@ -394,6 +395,7 @@ struct strangenessBuilder {
     } catch (...) {
       registry.fill(HIST("hCaughtExceptions"), 0.5f);
       LOG(error) << "Exception caught in DCA fitter process call!";
+      return false;
```

Comment:
Ah! now I get it

Reply:
Should we align also the other builders?

Reply:
Maybe! Done! Thanks! 👍